source/slang/hlsl.meta.slang

```diff
@@ -20248,3 +20248,23 @@ uint2 clock2x32ARB()
         };
     }
 }
+
+__glsl_version(450)
```

Comment:
```suggestion
__glsl_version(400)
```

Reply:
Since it is dependent on `GL_ARB_shader_clock`, should it not remain 450?